app/src/main/res/values/strings.xml

```diff
@@ -117,6 +117,9 @@
     <string name="give_feedback_screen_title">Give Feedback</string>
     <string name="report_bug_screen_title">Report Bug</string>
     <string name="debug_settings_screen_title">Debug Settings</string>
+    <string name="debug_settings_api_versioning_title">API VERSIONING</string>
```

Comment:
I would suggest to convert the string to uppercase in compose...Ee could miss this in other strings files

Reply:
This will only be shown for debug builds for now. But if in the future we decide to reuse it, I will make sure to upper case it in the corresponding composable.